whisper/whisperv5/filter.go

```diff
@@ -220,6 +220,10 @@ func matchSingleTopic(topic TopicType, bt []byte) bool {
 		bt = bt[:4]
 	}
 
+	if len(bt) < 4 {
```

Comment:
would be a good clean up if you will create a constant TopicLength and use it in this function

Reply:
it is already there, declared in the doc.go file

Reply:
done